autogpts/forge/forge/actions/registry.py

```diff
@@ -186,8 +188,11 @@ async def run_action(
 
 if __name__ == "__main__":
     import sys
+    import asyncio
```

Comment:
swifty removal and replacement with more amenable action to not actually having an agent seeded. list_files requires a full agent to be able to be pulled in to actually work